bug-1690385-pref-activity-stream-new-tab-default-content-release-85-87

```diff
@@ -0,0 +1,152 @@
+[experiment]
+end_date = '2020-04-13'
+
+segments = ["attributed", "non_attributed"]
+
+enrollment_period = 8
+
+reference_branch = "baseline-newtab-content"
+
+
+## data sources
+
+[data_sources]
+
+## data source: AS session pings data
+[data_sources.as_session_ping]
+from_expression = "(SELECT client_id, user_prefs, date(submission_timestamp) as submission_date FROM  `moz-fx-data-shared-prod.activity_stream.sessions`)"
+
+experiments_column_type = "native"
+
+
+
+
+## Segments
+
+[segments]
+
+[segments.attributed]
+select_expression = 'attr_source is not null'
+data_source = "npp"
+
+[segments.non_attributed]
+select_expression = 'attr_source is null'
+data_source = "npp"
+
+[segments.data_sources.npp]
+from_expression = """
+    (SELECT
+        DATE(submission_timestamp) AS submission_date,
+        client_id,
+        environment.settings.attribution.source AS attr_source
+    FROM `moz-fx-data-shared-prod.telemetry.new_profile`
+    WHERE DATE(submission_timestamp) BETWEEN "{{experiment.start_date_str}}" AND "{{experiment.last_enrollment_date_str}}"
+    AND environment.partner.distribution_id IS NULL
+    AND coalesce(environment.settings.attribution.ua, '') != 'firefox'
+    )
+    """
+
+
+## metrics
+
+[metrics]
+overall = ['days_of_use', 'search_count', 'organic_search_count', 'unenroll', 
+           'pocket_rec_clicks', 'pocket_spoc_clicks', 'homepage_search_count', 
+           'topsite_clicks', 'highlight_clicks', 'homepage_switch', 
+           'turned_off_search', 'turned_off_topsites', 'turned_off_pocket', 
+           'turned_off_highlights']
+
+
+## homepage_search_count
+
+[metrics.homepage_search_count]
+select_expression = "SUM(CASE WHEN source IN ('newtab','abouthome') THEN sap ELSE 0 END)"
+data_source = 'search_clients_daily'
+
+[metrics.homepage_search_count.statistics.bootstrap_mean]
+
+
+## topsite_clicks
+
+[metrics.topsite_clicks]
+select_expression = "COUNTIF(
+                event = 'CLICK'
+                AND source = 'TOP_SITES')"
```

Comment:
you can't have newlines in double-quoted strings unless you backslash-escape them; https://toml.io/en/v1.0.0#string

Try triple-quotes, just like Python:

```toml
select_expression = """COUNTIF(
                event = 'CLICK'
                AND source = 'TOP_SITES')"""
```

here and below

Reply:
good catch, +1